### ISSUE TITLE
Dynamically change style (background color) of input field on button click

### ISSUE THREAD

**Issue by darlontrofy** [formly-js/ngx-formly#1132]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x ] support request
```

**Current behavior**

When I added the code below to change the color of an input field when a tag input is clicked, the color of the input field is not changed, what else do I need?

addonRight: {
              class: 'fa fa-code',
               onClick: (to, fieldType, $event) => {
                    this.showInputSelector(to, fieldType, $event);
                },
        }   

showInputSelector(to, fieldType, $event) {
    
       fieldType.field.templateOptions.attributes.style = "background: green; border-color: green;";
       to.placeholder = "test";
        $event.stopPropagation();
    }
**Expected behavior**

Update color of input field
**Minimal reproduction of the problem with instructions**

https://stackblitz.com/edit/angular-objr91
**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

Angular 4
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by aitboudad** [comment]
fixed in `5.0.0-beta.3`, in order to take account of attributes you have to override it:
```ts
fieldType.field.templateOptions.attributes = {
  style: "background: green; border-color: green;"
};
```

**Comment by darlontrofy** [comment]
Thanks..works very well.
